templates/options.conf.erb

```diff
@@ -4,3 +4,6 @@ forwarders { <%= scope.lookupvar('::dns::forwarders').join("; ") %>; };
 <% end -%>
 
 listen-on-v6 { <%= scope.lookupvar('::dns::listen_on_v6') %>; };
+<% if (@operatingsystem == 'FreeBSD') -%>
```

Comment:
argh, sorry... it's too early. The package is coming with a config file that's including this line so it's working out of the box. If this is a real problem, I can try to contact the package maintainer.